doc/setup.md

```diff
@@ -181,20 +181,79 @@ To get started using the application with docker,
-5. Now you should be able to run `make start`. This will start the application in daemon mode, which means that the server will keep running in the background. If you navigate to  `localhost:3000` in your browser, you will see an error. This is normal, and it means that you still need to setup the database.
-6. To setup the database, you can run
-  `SYSTEM_EMAIL=${SYSTEM_EMAIL} SYSTEM_PASSWORD=${SYSTEM_PASSWORD} make seed`
-  This will setup the database and create a default admin user with the email and password as specified by the `SYSTEM_EMAIL` and `SYSTEM_PASSWORD` environment variables which are ultimately passed to `docker-compose` with the `-e` option. If you don't want to create the default user, you can just run `make prepare` and create the account using the sign up option on the website.
 
-7. You should now be able to reload `localhost:3000` in your browser. If everything went well, the website should appear and be functional. You can sign in using the email and password you set in the previous step. This docker compose also setups an a `mailcatcher` server, which you can access at `localhost:1080`. All emails will be delivered to mailcatcher, which should allow you to setup user accounts.
+### Bootstrapping the application for development
 
-**NOTE** Do not use this method in production! This is for **testing & development only** the configuration used with in this docker-compose file is highly insecure and should never be exposed to the public internet.
+```
+bin/bootstrap
+```
+
+Tears down any exiting images, containers, networks, etc. and sets everything up from scratch. This also seeds the development database - you will be prompted to enter your desired credentials for the development admin user.
+
+### Running tests
+
+```
+bin/test-docker
+```
+
+Runs the tests inside the context of the `app` container
+
+Analogous to: `bin/test`
+
+### Starting the application and its services
+
+```
+bin/start
+```
+
+Starts the web application container and its supporting services - RDBMS, mailcatcher, etc.
+
+Analogous to: `bin/rails server`
+
+The application should be available at [https://localhost:3000](http://localhost:3000). You can sign in using the email and password you provided in the when you ran `bin/bootstrap`. This docker compose also setups an a `mailcatcher` server, which you can access at `localhost:1080`. All emails will be delivered to mailcatcher, which should allow you to setup user accounts.
+
+**NOTE** It's probably a good idea to start the `webpack-dev-sever` to compile webpacks out-of-band of webserver requests - this tends to run into the `rack-timeout` in the docker development environment. The `bin/start` script takes an optional service name - or list of names - to startup. The `webpack-dev-server` _can_ be started issuing the following command:
+
+```
+bin/start webpacker
```

Comment:
We _could_ also make the `app` service depend on the `webpacker` - in this way everything would be started auto-magically on a `bin/start` or `bin/start app`. The problem is that we will also start the `webpacker` service in contexts where it's probably not so helpful `bin/shell` and `bin/console` for example.

Reply:
Addressed by: https://github.com/rubyforgood/mutual-aid/pull/715/commits/dde1d54df645dc4299f25cf4ee37c4acc5b2c2ca